src/apps/intel_mp/README.md

```diff
@@ -72,18 +72,19 @@ For a given NIC, all driver instances should have this parameter either
 enabled or disabled uniformly. If this is enabled, *macaddr* must be
 specified.
 
-— Key **vmdq_queuing_mode**
+— Key **vmdq_queueing_mode**
```

Comment:
This is pretty minor, but apparently "queuing" and "queueing" are both valid spellings so I'd prefer not to change it to avoid breaking any existing uses.

Reply:
While the change is backwards compatible (accepts both spellings), a quick internet search indicates that “queuing” is actually the more common spelling nowadays. Seems I have trusted my spell-checker too much. I suppose I will revert this for simplicity.